app/background.ts

```diff
@@ -59,6 +67,7 @@ class BackgroundWorker {
     const tabRecords = this.data[this.currentTab];
     if (tabRecords.enabled && this.currentTab === details.tabId) {
       tabRecords.requests.push(details);
+      tabRecords.count += 1 
```

Comment:
why keep this separate? just take the `tabRecords.requests.length`.

Reply:
Yeah, thats better